### PR TITLE
Merge 0.23 to main

### PR DESCRIPTION
**Pull request by rossabaker**

### CODE REVIEW

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+changelog:
+  exclude:
+    labels:
+      - series/0.22
+      # - series/0.23
```
[the remark this answers is not in the document]
Comment (rossabaker) [reply .github/release.yml:5]: You'd think I'd have learned after the last one.

File: website/src/hugo/content/changelog.md [hunk 2]
```diff
@@ -25,27 +25,6 @@ This milestone is the first to include Scala 3 support for http4s-scalatags.
...
-	* Non-trivial merge to main by @rossabaker in https://github.com/http4s/http4s/pull/5530
-	* Merge branch `series/0.23` into `main` by @danicheg in https://github.com/http4s/http4s/pull/5664
-	* Unpin `scalatags` in the Steward config by @danicheg in https://github.com/http4s/http4s/pull/5668
-	* Merge 0.23 -> main by @armanbilge in https://github.com/http4s/http4s/pull/5673
-
-* New Contributors
-	* @jenshalm made their first contribution in https://github.com/http4s/http4s/pull/5313
-	* @barshirtcliff made their first contribution in https://github.com/http4s/http4s/pull/5513
-
-**Full Changelog**: https://github.com/http4s/http4s/compare/v1.0.0-M29...v1.0.0-M30
```
Comment (armanbilge) [inline website/src/hugo/content/changelog.md:48]: Is this supposed to be removed?

File: website/src/hugo/content/changelog.md [hunk 3]
```diff
@@ -25,27 +25,6 @@ This milestone is the first to include Scala 3 support for http4s-scalatags.
...
 	* Migrate Website to Laika by @jenshalm in https://github.com/http4s/http4s/pull/5313
 	* Update website with SJS on 0.23, http4s-dom project by @armanbilge in https://github.com/http4s/http4s/pull/5453
-	* Fix website logo by @armanbilge in https://github.com/http4s/http4s/pull/5459
-	* Update all the urls to new http4s-dom site by @armanbilge in https://github.com/http4s/http4s/pull/5507
-	* Fix markdown in readme by @bplommer in https://github.com/http4s/http4s/pull/5523
-	* updated getting-help.md to point to discord by @barshirtcliff in https://github.com/http4s/http4s/pull/5513
-	* EPUB download of the docs by @danicheg in https://github.com/http4s/http4s/pull/5640
-	* website: add favicon by @jenshalm in https://github.com/http4s/http4s/pull/5656
-
-* Behind the scenes
```
Comment (danicheg) [inline website/src/hugo/content/changelog.md:35]: Probably we shouldn't delete this? Some

Comment (danicheg) [reply website/src/hugo/content/changelog.md:35]: Ouch, @armanbilge has pointed it already.

File: website/src/hugo/content/changelog.md (hunk 2 above)
Comment (rossabaker) [reply website/src/hugo/content/changelog.md:48]: This is what I get for merging during a corporate training.

**Review by armanbilge — APPROVED** [review]

**Comment by danicheg** [comment]
"1 check passed" seems a bit odd 😮

**Comment by armanbilge** [comment]
Yes, it was a `[ci skip]` commit. But the ones above it looked good, just flakes. Main looks green rn.

**Comment by danicheg** [comment]
Wow, nice workaround, good to know :)

**Pull request merged**